doc/releases/changelog-0.35.0.md

```diff
@@ -236,6 +365,9 @@
 * A function called `apply_operation` has been added to the new `qutrit_mixed` module found in `qml.devices` that applies operations to device-compatible states.
```

Comment:
Just putting a note here that the above three qutrit entries can be combined into one